pyproject.toml

```
@@ -8,7 +8,7 @@ authors = ["Nyaruka <code@nyaruka.com>"]
 repository = "http://github.com/rapidpro/rapidpro"
 
 [tool.poetry.dependencies]
-python = "^3.10"
+python = "^3.11"
```

rapidpro seems to work just fine on python-3.10
```suggestion
python = ">=3.10<4"
```

---

Currently yes but we'd like to start using newer Python features without worrying if they're supported on older versions so the goal is to get the required version to 3.12